jupyterlab_gitsync/src/components/toolbar_buttons.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { classes, style } from 'typestyle';
+import {
+  toolbarItemClass,
+  toolbarButtonClass,
+  toolbarButtonIconClass,
+} from '../style/toolbar';
+
+import { Props } from './panel';
+
+export interface State {
+  state: string;
+  label: string;
+  style: ReturnType<typeof style>;
+}
+
+class ToolbarButton extends React.Component<Props, State> {
+  constructor(
+    props: Props,
+    init_label: string,
+    init_style: ReturnType<typeof style>
+  ) {
+    super(props);
+    this.state = {
+      state: 'Running',
+      label: init_label,
+      style: init_style,
+    };
+  }
+
+  render(): React.ReactElement {
+    return (
+      <div
+        className={classes(
+          'jp-ToolbarButton',
+          'jp-Toolbar-item',
+          toolbarItemClass
+        )}
+      >
+        <button
+          className={classes(
+            'bp3-button',
+            'bp3-minimal',
+            'jp-ToolbarButtonComponent',
+            'minimal',
+            'jp-Button',
+            toolbarButtonClass
+          )}
+          title={this.state.label}
+          onClick={this._onClick}
+        >
+          <span className={classes('bp3-button-text')}>
+            <span
+              className={classes(
+                'jp-ToolbarButtonComponent-icon',
+                toolbarButtonIconClass,
+                this.state.style
+              )}
+            >
+              {' '}
+            </span>
+          </span>
+        </button>
+      </div>
+    );
+  }
+
+  protected _onClick = (): void => {
+    console.log('hi');
```

Comment:
Please try to remove all console.log statements that do not provide real debugging information before submitting for a PR. Obviously it's useful to have this for debugging purposes during development but sometimes it's easy to forget to remove them before a release.

Reply:
This line has been changed and will be pushed in a future PR so that there is no console.log statement.